api/docker/entrypoint.sh

```diff
@@ -8,8 +8,16 @@ if [[ "${MIGRATION_ENABLED}" == "true" ]]; then
 fi
 
 if [[ "${MODE}" == "worker" ]]; then
-  celery -A app.celery worker -P ${CELERY_WORKER_CLASS:-gevent} -c ${CELERY_WORKER_AMOUNT:-1} --loglevel INFO \
-    -Q ${CELERY_QUEUES:-dataset,generation,mail,ops_trace,app_deletion}
+  if [ "${CELERY_AUTO_SCALE,,}" = "true" ]; then
+    MAX_WORKERS=${CELERY_MAX_WORKERS:-4}
```

Comment:
We could set the max workers by using `nproc` to identify the number available CPU cores, as celery worker command itself suggested.